src/azure-cli-core/azure/cli/core/breaking_change.py

```diff
@@ -13,7 +13,7 @@
 
 logger = get_logger()
 
-NEXT_BREAKING_CHANGE_RELEASE = '2.67.0'
+NEXT_BREAKING_CHANGE_RELEASE = '2.73.0'
```

Comment:
We use `2.74` in this PR.
Please double confirm the version number.

Reply:
I reached out to Yong regarding the next breaking change version number and received `2.73.0` as the response. I'm flexible with either `2.73.0` or `2.74.0`, as I plan to update to the confirmed version as soon as it's finalized.

Reply:
Find `2.73.0` in https://learn.microsoft.com/en-us/cli/azure/azure-cli-support-lifecycle too.
Hmm, #30091 might be wrong.

Reply:
`2.54.0` is 2023 ignite version, `2.61.0` is 2024 build version. Weird.

Reply:
I got 2.74 from @dcaro: https://github.com/Azure/azure-cli/pull/30091#discussion_r1819649786.

Reply:
I think it's acceptable to merge this PR for now. We can create another PR once we have the finalized version.